docs/modules/user-guide/partials/proc_deploying-a-devfile-registry-with-olm.adoc

```diff
@@ -0,0 +1,62 @@
+[id="deploying-a-devfile-registry-with-operator-lifecycle-manager_{context}"]
+= Deploying a devfile registry with operator lifecycle manager
+
+[role="_abstract"]
+When you have Operator Lifecycle Manager (OLM) installed on your cluster, use the devfile registry operator to install and deploy a devfile registry.
+
+.Procedure
+..Installing the devfile registry operator
+. Install link:https://github.com/operator-framework/operator-lifecycle-manager[OLM].
+. Download and install the link:https://mirror.openshift.com/pub/openshift-v4/clients/operator-sdk/latest/[operator-sdk CLI].
+. Run the `operator-sdk CLI` command to install the `devfile-registry operator`:
```

Comment:
Is there a reason `devfile-registry operator` is formatted in monospace (and written differently) here? Should this just be "devfile registry Operator" like the previous instances?